InMobiAdapter/src/main/java/com/chartboost/mediation/inmobiadapter/InMobiAdapter.kt

```diff
@@ -384,6 +369,41 @@ class InMobiAdapter : PartnerAdapter {
         }
     }
 
+    /**
+     * Build a [JSONObject] that will be passed to the InMobi SDK for GDPR during [setUp].
+     *
+     * @param gdprConsent A Boolean indicating whether GDPR consent is granted or not.
+     *
+     * @return a [JSONObject] object as to whether GDPR  .
+     */
+    private fun buildGdprJsonObject(gdprConsent: Boolean, context: Context): JSONObject {
+        return JSONObject().apply {
+            try {
+                put(InMobiSdk.IM_GDPR_CONSENT_AVAILABLE, gdprConsent)
+                put("gdpr", if (gdprApplies == true) "1" else "0")
+                put(InMobiSdk.IM_GDPR_CONSENT_IAB, getTCString(context))
```

Comment:
should we still add it if it's empty?

Reply:
I was curious about this as well. Perhaps, we can make the `getTCFString` method return a null, then add the string if it exist and ignore if we don't find it. 🤔

Would that be alright? 😃

Reply:
can also take if not empty

Reply:
Ah. True.

Reply:
Pushed a new commit, @bichenwang. I just need another check ✔️